app/services/invite_user.rb

```diff
@@ -15,8 +15,7 @@ def call
     set_invitation_organisations
     set_invitation
     result.invitation = @invitation
-    return if invitation_already_sent_today? && !@invitation.format_postal?
-
+    check_if_invitation_should_be_sent
```

Comment:
```suggestion
    check_if_invitation_should_be_sent!
```
pour être plus cohérent avec ce qu'on fait par ailleurs